### PR TITLE
Refactor CSS files in src/screens/OrganizationFundCampaign 

### LINKED ISSUE #2515: Refactor CSS files in src/screens/OrganizationFundCampaign
# Refactor CSS files in src/screens/OrganizationFundCampaign

To improve the UI/UX of Talawa-Admin we have been working on:
1. making the application more suitable to color blind users
2. streamlining all CSS into a single global file

## Additional information:
1. The foundational work for this was completed in this PR:
    - https://github.com/PalisadoesFoundation/talawa-admin/pull/2466
2. The desired single CSS file is:
    - `src/style/app.module.css`

## The challenge:
The goal is to convert the CSS file in this subdirectory and all the components related to this screen to use this new design pattern. This means that:
1. All CSS files in this subdirectory and all the CSS files in related components must be merged into the global CSS file using pre-existing CSS.
2. All CSS files in this subdirectory and all the CSS files in related components must be deleted after the merge.
3. The remaining relevant files in this subdirectory must only reference the global CSS file.
4. All the CSS files in the related components of this subdirectory must adhere to the same guidelines.

### PR DESCRIPTION
**Pull request by mandeepnh5**
**What kind of change does this PR introduce?**
This PR will Refactor CSS files in src/screens/OrganizationFundCampaign 

**Issue Number:**
Fixes #2515 

**Did you add tests for your changes?**
No

**If relevant, did you update the documentation?**
No

**Does this PR introduce a breaking change?**
No

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-admin/blob/master/CONTRIBUTING.md)?**
Yes

## Summary by CodeRabbit

- **New Features**
	- Introduced new styling classes for the Organization Fund Campaign feature, enhancing layout and responsiveness.
	- Updated button styles for campaign submission and management.

- **Bug Fixes**
	- Improved specificity in class naming for better clarity and maintainability.

- **Chores**
	- Removed outdated CSS file to streamline styling references.

### CODE REVIEW

**Comment by palisadoes** [comment]
Please fix the failing tests

File: src/style/app.module.css [hunk 1]
```diff
@@ -992,6 +992,181 @@ hr {
...
+}
+
+.inputFieldOrganizationFundCampaign {
+  background-color: white;
+  box-shadow: 0 1px 1px #31bb6b;
+}
+
+.dropdownOrganizationFundCampaign {
+  background-color: white;
+  border: 1px solid #31bb6b;
```
Comment (palisadoes) [inline src/style/app.module.css:1078]: 1. We are migrating away from the green theme as part of our support for color blind users.
2. Take cues from other similar elements in this file and the PR mentioned in the issue.
    1. https://github.com/PalisadoesFoundation/talawa-admin/pull/2466

File: src/style/app.module.css [hunk 2]
```diff
@@ -992,6 +992,181 @@ hr {
...
+  position: relative;
+  display: inline-block;
+  color: #31bb6b;
```
Comment (palisadoes) [inline src/style/app.module.css:1081]: 1. We are migrating away from the green theme as part of our support for color blind users.
2. Take cues from other similar elements in this file and the PR mentioned in the issue.
    1. https://github.com/PalisadoesFoundation/talawa-admin/pull/2466

File: src/style/app.module.css [hunk 3]
```diff
@@ -992,6 +992,181 @@ hr {
...
+  cursor: pointer;
+}
+.campaignNameInfo:hover {
+  color: blue;
+  transform: translateY(-2px);
+}
+
+.inputFieldOrganizationFundCampaign {
+  background-color: white;
+  box-shadow: 0 1px 1px #31bb6b;
```
Comment (palisadoes) [inline src/style/app.module.css:1073]: 1. We are migrating away from the green theme as part of our support for color blind users.
2. Take cues from other similar elements in this file and the PR mentioned in the issue.
    1. https://github.com/PalisadoesFoundation/talawa-admin/pull/2466

**Review by palisadoes — CHANGES_REQUESTED** [review]
Please refer to the use of color:
- #31bb6b

**Comment by palisadoes** [comment]
Please fix the failing tests

**Comment by mandeepnh5** [comment]
ok I forgot i will fix the colors now

**Review by palisadoes — APPROVED** [review]

**Pull request merged**